plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -3296,6 +3296,10 @@ else if (isFullAcidTable(table.getParameters())) {
                 .map(Column::getName)
                 .forEach(partitioningColumns::add);
 
+        // For transactional bucketed tables we don't want to split output files therefore we need to have single writer
+        // per partition.
+        boolean multipleWritersPerPartitionSupported = !isTransactionalTable(table.getParameters());
```

Comment:
this looks like a pre-exiting bug (and quite important one, potential data corruption). Would it be possible to fix it separately + add a test?

Reply:
this is not an existing as we discussed offline